src/App.tsx

```diff
@@ -15,71 +15,76 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selected: string | null;
+};
+export class App extends Component<{}, State> {
+  state = {
+    selected: 'Jam',
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  render() {
+    const { selected } = this.state;
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+    return (
+      <main className="section container">
+        {/* <h1 className="title">No goods selected</h1> */}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+        {selected
+          ? (
+            <h1 className="title is-flex is-align-items-center">
+              {`${selected} is selected`}
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => {
+                  this.setState({ selected: null });
+                }}
+              />
+            </h1>
+          )
+          : (<h1 className="title">No goods selected</h1>)}
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+        <table className="table">
+          <tbody>
+            {goods.map(selectedGood => (
+              <tr
+                data-cy="Good"
+                className={selected === selectedGood
```

Comment:
Move this check to the variable & reuse it, DRY
```suggestion
                className={...
```